package.json

```diff
@@ -102,6 +102,7 @@
     "node-stream-zip": "^1.15.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-markdown": "^9.0.3",
```

Comment:
Let's look for a smaller dependency.

Reply:
After briefly trying to adapt https://github.com/developit/snarkdown to write our own React component for this, I gave up and used https://github.com/quantizor/markdown-to-jsx instead. It's 6kb gzipped.